editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -2110,6 +2110,19 @@ void SpatialEditorViewport::_notification(int p_what) {
 
 		_update_freelook(delta);
 
+		if (cinema_mode->is_pressed()) {
+			Camera *cam = editor->get_scene_tree_dock()->get_editor_data()->get_edited_scene_root()->get_viewport()->get_camera();
+			if (cam != NULL && cam != previewing)
+			{
```

Comment:
There's a style issue with this brace style: https://travis-ci.org/godotengine/godot/jobs/324873663

See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html